geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractIncrByFloatIntegrationTest.java

```diff
@@ -30,15 +31,16 @@
 
 public abstract class AbstractIncrByFloatIntegrationTest implements RedisPortSupplier {
 
+  private static final int JEDIS_TIMEOUT =
+      Math.toIntExact(GeodeAwaitility.getTimeout().toMillis());
+
   private Jedis jedis;
-  private Random rand;
   private static final int REDIS_CLIENT_TIMEOUT =
       Math.toIntExact(GeodeAwaitility.getTimeout().toMillis());
```

Comment:
Seems like this might've gotten duplicated in the merge.

Reply:
Thanks